core/tests/protractor.conf.js

```diff
@@ -91,6 +91,9 @@ exports.config = {
 
     i18n: [
       'protractor/i18n.js'
+    ],
+    tour: [
```

Comment:
Put this in the misc suite. Otherwise we'll have to add yet another suite to Travis...